ext/telemetry/errors.js

```diff
@@ -4,13 +4,14 @@ module.exports = function(tracker) {
   return {
     trackUncaught: () => {
       window.onerror = (msg, url, line, col, err) => {
-        var data = {
-          msg: msg,
-          url: url,
-          line: line,
-          col: col,
-        };
-        tracker.trackEvent("uncaught error", { uncaughtError: data });
+        tracker.trackEvent("uncaught error", {
+          uncaughtError: {
```

Comment:
Ok, so I commented on an outdated commit, so I'll make the comment here again.

Using shorthand object literal is better in this case:
```js
uncaughtError: {msg, url, line, col}
```